observations/obs_converters/even_sphere/even_sphere.m

```diff
@@ -66,99 +88,99 @@
-         fprintf(fid, '%6.2f\n', dlat(hloc));
-         % Now the date and time
-         fprintf(fid, '%5i %3i %3i %3i %2i %2i \n', year, month, day, hour, 0, 0);
-         % Finally, the error variance, 1 for temperature, 4 for winds
-         if(field == 1)
-            fprintf(fid, '%2i\n', 1);
-         else
-            fprintf(fid, '%2i\n', 4);
-         end
-     
-         % Need value and qc for testing
-         if(diagnostic_test)
-            fprintf(fid, '%2i\n', 1);
-            fprintf(fid, '%2i\n', 0);
-         end
-      end   
-   end
-end
-
-% File name
-fprintf(fid, 'set_def.out\n');
-
-% Done with output, close file
-fclose(fid);
-
-
+ % Need to generate output for driving create_obs_sequence
+ fid = fopen('even_create_input', 'w');
+
+ % Output the total number of observations
+ fprintf(fid, '%6i\n', num_obs);
+ % 0 copies of data
+ if(diagnostic_test) 
+    fprintf(fid, '%2i\n', 1);
+ else
+    fprintf(fid, '%2i\n', 0);
+ end
+ 
+ % 0 QC fields
+ if(diagnostic_test) 
+    fprintf(fid, '%2i\n', 1);
+ else
+    fprintf(fid, '%2i\n', 0);
+ end
+ 
+ % Metadata for data copy
+ if(diagnostic_test)
+    fprintf(fid, '"observations"\n');
+ end
+ % Metadata for qc
+ if(diagnostic_test)
+    fprintf(fid, '"Quality Control"\n');
+ end
+ 
+ 
+ % Loop to create each observation
+ for hloc = 1:n
+    for vloc = 1:size(levels, 2)
+       for field = 1:3
+          % 0 indicates that there is another observation; 
+          fprintf(fid, '%2i\n', 0);
+          % Specify obs kind by string
+          if(field == 1)
+             fprintf(fid, 'RADIOSONDE_TEMPERATURE\n');
+          elseif(field == 2)
+             fprintf(fid, 'RADIOSONDE_U_WIND_COMPONENT\n');
+          elseif(field == 3)
+             fprintf(fid, 'RADIOSONDE_V_WIND_COMPONENT\n');
+          end
+          % Select pressure as the vertical coordinate
+          fprintf(fid, '%2i\n', 2);
+          % The vertical pressure level
+          fprintf(fid, '%5i\n', levels(vloc));
+          % Lon and lat in degrees next
+          fprintf(fid, '%6.2f\n', dlon(hloc));
+          fprintf(fid, '%6.2f\n', dlat(hloc));
+          % Now the date and time
+          fprintf(fid, '%5i %3i %3i %3i %2i %2i \n', year, month, day, hour, 0, 0);
+          % Finally, the error variance
+          if(field == 1)
+             fprintf(fid, '%6.2f\n', T_variance(vloc));
+          else
+             fprintf(fid, '%6.2f\n', W_variance(vloc));
+          end
+      
+          % Need value and qc for testing
+          if(diagnostic_test)
+             fprintf(fid, '%2i\n', 1);
+             fprintf(fid, '%2i\n', 0);
+          end
+       end   
+    end
+ end
+ 
+ % File name
+ fprintf(fid, 'set_def.out\n');
+ 
+ % Done with output, close file
+ fclose(fid);
 
 % Some plotting to visualize this observation set
 plot(lon, lat, '*');
 hold on
 
-% Plot an approximate CAM T85 grid for comparison
-% 256x128 points on A-grid
+% Plot an approximate CAM grid for comparison
```

Comment:
not sure you want to inject so much CAM specific stuff here.  this is also used for Bgrid tests.

Reply:
CAM references have been removed.